crypto/src/mls/conversation/decrypt.rs

```diff
@@ -189,17 +189,22 @@ impl MlsCentral {
         conversation_id: &ConversationId,
         message: impl AsRef<[u8]>,
     ) -> CryptoResult<MlsConversationDecryptMessage> {
-        let decrypt_message = Self::get_conversation_mut(&mut self.mls_groups, conversation_id)?
+        let decrypt_message = self
+            .get_conversation(conversation_id)
+            .await?
+            .write()
+            .await
             .decrypt_message(
                 message.as_ref(),
                 &self.mls_backend,
                 self.callbacks.as_ref().map(|boxed| boxed.as_ref()),
             )
             .await?;
         if !decrypt_message.is_active {
-            self.mls_groups
-                .remove(conversation_id)
-                .ok_or(CryptoError::ImplementationError)?;
+            self.wipe_conversation(conversation_id).await?;
```

Comment:
Yes it's probably better this way as it does not leave a ghost conversation in the keystore. Maybe amend the documentation to let users know conversation could get wiped implicitly.

Reply:
Yeah, I'll need to document this change for sure - I was having the case where the GroupStore was working *too well* and pruning it in cache would cause it to function as usual (because the fallback mechanism of not having it in cache would fetch it in the keystore)

There's an alternative possible, is to set the group to active = false & not wipe it implicitely, I believe we have the guards everywhere to not perform actions on inactive groups.

Reply:
I'd be in favor of the alternative option: just mark the group inactive and not delete anything either on the keystore or in cache. openmls is supposed to provide us the guards that an inactive group cannot be operated